### ISSUE TITLE
Refresh 404 error with routes

### ISSUE THREAD

**Issue by dave-schneider-ila** [johnpapa/angular2-go#24]
Is there a clean way to handle the 404 error when you refresh the browser manually?

**Comment by lazyspark** [comment]
In the routes you can have the route ** go to a component that will have a 404 template. 

```
{ path: '**', pathMatch:'full', redirectTo: '/randompath' } 
```

**Comment by johnpapa** [comment]
yep.

**Comment by lazyspark** [comment]
@johnpapa would you like me to send pull request with the change / it would my first not sure if this is right place

**Comment by johnpapa** [comment]
sure. bu look at the event-view repo. it has the route, which is easy. but use the PageNotFoundCOmponent in that repo for this too. please :)

**Comment by lazyspark** [comment]
Thanks for the opportunity. Pull request #28 submitted, Hopefully it meets the required guidelines

**Comment by johnpapa** [comment]
closed by #28 

Thanks for contributing!
